README.md

```diff
@@ -1,8 +1,30 @@
 # Northcoders News API
 
+Project Summary: This is a NEWS API with endpoints to access news articles, users and comments on articles.
+It is designed to be able to access app data programmatically in order to provide this data to front-end architecture.
+
+Link to hosted repo: https://my-nc-news-s3ph.onrender.com/
+Available endpoints can be found at: https://my-nc-news-s3ph.onrender.com/api
+Eg: https://my-nc-news-s3ph.onrender.com/api/articles
+
+This project requires that you have downloaded and minimum version of
+• Node.js V14
+• PostgreSQL V10
+
+Step 1:
 In order to clone this repo and connect databses locally, please create two .env files:
```

Comment:
You don't need the .env files to be able to clone the repo - they need to be added once the repo has been cloned 🙂